spec/features/navigation_bar_spec.rb

```diff
@@ -1,17 +1,23 @@
 require 'rails_helper'
 feature "Navigation bar" do 
-  scenario "has link sign in and home link when user is not signed on" do 
+  before(:example) do |example|
+    unless example.metadata[:skip_before]
```

Comment:
`:skip_before` would be more accurately named with something like `:skip_sign_in`. This will make it clearer to developers reading tests further down the file what it happening when they see the `:skip_...` metadata option.